modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -25,6 +25,18 @@ Ensure that the host project applies one of the following configurations to the
 * `roles/compute.securityAdmin`
 ====
 
+.Required permissions for granting the service account the required permissions in the host project
```

Comment:
The line 18 tells "_Ensure that the host project applies **one of the following configurations** to the service account:_", which lists 3 following sections, 
- Required permissions for creating firewalls in the host project
- Required permissions for granting the service account the required permissions in the host project
- Required minimal permissions

In fact, the 2nd section applies, for both the 1st and the 3rd. 

How about re-org like below? 
------
Ensure that the host project applies one of the following configurations to the service account:

- Required permissions for creating firewalls in the host project
- Required permissions for without creating firewalls in the host project

Besides, if you do not supply a service account for control plane nodes in the `install-config.yaml` file, please also grant below permissions to the service account in the host project: 
* `resourcemanager.projects.getIamPolicy`
* `resourcemanager.projects.setIamPolicy`

Reply:
@barbacbd Had you got a chance reviewing this comment and the last comment? Thanks!